src/ProgramBlockEditor.test.js

```diff
@@ -148,7 +149,17 @@ function getRunButton(programBlockEditorWrapper) {
 }
 
 function getAddNodeButtonAtPosition(programBlockEditorWrapper, index: number) {
-    const addNodeButton = programBlockEditorWrapper.find('.AddNode__expanded-button');
+    const addNodeButton = programBlockEditorWrapper.find(AriaDisablingButton).filter('.AddNode__expanded-button');
+    return addNodeButton.at(index);
+}
+
+function getEnabledAddNodeCount(programBlockEditorWrapper) {
+    const addNodeButton = programBlockEditorWrapper.find(AriaDisablingButton).filter('.AddNode__expanded-button');
+    return addNodeButton.length;
+}
+
+function getAddNodeExpandButton(programBlockEditorWrapper) {
```

Comment:
Rename to `getExpandAddNodeToggleSwitch()`

Reply:
Done.